changelogs/8.9.asciidoc

```diff
@@ -0,0 +1,35 @@
+[[release-notes-8.9]]
+== APM version 8.9
+
+https://github.com/elastic/apm-server/compare/8.9\...main[View commits]
+
+* <<release-notes-8.9.0>>
+
+[float]
+[[release-notes-8.9.0]]
+=== APM version 8.9.0
+
+[float]
+==== Breaking Changes
+
+[float]
+==== Deprecations
```

Comment:
Are we expecting additions to these sections? If not, let's remove them.

Reply:
Done! Thanks!